factfeel_client.py

```diff
@@ -16,6 +20,17 @@ def __init__(self, ip = None, lights = None, colors = None, spectrum_size = 11):
         
         self.spectrum_size = spectrum_size
         
+        
+        ######
```

Comment:
I added documentation here to help indicate the converter's behavior with multiple calls to the convert funtions.